src/communication/xcConnection.ts

```diff
@@ -34,7 +29,9 @@ export class DefaultConnection implements Connection {
             log.debug("getModel request failed");
             log.debug(err);
         };
-        session.init(openListener, errorListener);
+        let closeListener = (_: CloseEvent) => {
```

Comment:
how does the user know there was an error? 
as far as I understand the callback is never called.